cinema/views.py

```diff
@@ -0,0 +1,52 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework.request import Request
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def get_list(request: Request) -> Response:
```

Comment:
name is not really descriptive, fix that
You should specify what list we get here. Same for detail page. We'll have more views later it's important to name them properly